src/main/java/edu/greenblitz/robotName/utils/Conversions.java

```diff
@@ -23,33 +24,33 @@ public static double revolutionsToMeters(double revolutions) {
             return revolutions * MK4iSwerveConstants.WHEEL_CIRC;
         }
         public static double convertRadsToTicks(double angInRads) {
-            return angInRads / MK4iSwerveConstants.angleTicksToRadians;
+            return angInRads / MK4iSwerveConstants.ANGLE_TICKS_TO_RADIANS;
         }
 
         public static double convertTicksToRads(double angInTicks) {
-            return angInTicks * MK4iSwerveConstants.angleTicksToRadians;
+            return angInTicks * MK4iSwerveConstants.ANGLE_TICKS_TO_RADIANS;
         }
 
         public static double convertMetersToTicks(double distanceInMeters) {
-            return distanceInMeters / MK4iSwerveConstants.linTicksToMeters;
+            return distanceInMeters / MK4iSwerveConstants.LIN_TICKS_TO_METERS;
```

Comment:
What does Local Interconnect Network has to do with our robot code. (https://en.wikipedia.org/wiki/Local_Interconnect_Network)